msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -58,13 +58,13 @@ To add tabs to an existing meeting:
 
 # [Desktop](#tab/desktop)
 
-:::image type="content" source="~/assets/images/apps-in-meetings/PreMeetingTab.png" alt-text="Tabs during a meeting.":::
+:::image type="content" source="~/assets/images/apps-in-meetings/PreMeetingTab.png" alt-text="The screenshot describes how to add tabs during a meeting in desktop.":::
 
 # [Mobile](#tab/mobile)
 
 After adding the tabs to an existing meeting on mobile, you can see the same apps in pre-meeting experience under **More** section of the meeting details.
 
-<img src="../assets/images/apps-in-meetings/mobilePostMeeting.png" alt="Mobile pre-meeting experience" width="200"/>  
+:::image type="content" source="../assets/images/apps-in-meetings/mobilePostMeeting.png" alt-text="Mobile pre-meeting experience.":::
```

Comment:
Alt text does not "describe" what's in the screenshot. Also, is there a conflicting image name? The screenshot name contains postmeeting but alt-text says pre-meeting.

Reply:
Resolved